src/components/benefitsection/BenefitsSection.js

```diff
@@ -1,8 +1,16 @@
 import React, { useState } from "react";
```

Comment:
If you are not using the **React** reference any where in the code then why have you imported it. What are you smoking recently?

Reply:
Hey @Sehar-Saleem I mistakenly imported React while creating the components using a shortcut. I've resolved this and pushed the updated code. Thanks for pointing it out!